README.md

```diff
@@ -1,25 +1,24 @@
-# KubeKey
+<div align=center><img src="docs/img/kubekey-logo.svg?raw=true"></div>
 
 [![CI](https://github.com/kubesphere/kubekey/workflows/CI/badge.svg?branch=master&event=push)](https://github.com/kubesphere/kubekey/actions?query=event%3Apush+branch%3Amaster+workflow%3ACI+)
 
 > English | [中文](README_zh-CN.md)
 
-Since v3.0.0, [KubeSphere](https://kubesphere.io) changes the ansible-based installer to the new installer called KubeKey that is developed in Go language. With KubeKey, you can install Kubernetes and KubeSphere separately or as a whole easily, efficiently and flexibly.
+### 👋 Welcome to KubeKey!
 
-There are three scenarios to use KubeKey.
+KubeKey is an open source lightweight tool for deploying kubernetes clusters. It provides a flexible, rapid and convenient way to install Kubernetes/k3s only, both Kubernetes/k3s and KubeSphere, and related cloud-native add-ons. It is also an efficient tool to scale and upgrade your cluster.
+
+In addition, KubeKey also supports customized Air-Gap package, which is convenient for users to quickly deploy clusters in offline environment.
 
-* Install Kubernetes only
-* Install Kubernetes and KubeSphere together in one command
-* Install Kubernetes first, then deploy KubeSphere on it using [ks-installer](https://github.com/kubesphere/ks-installer)
+> KubeKey has passed [CNCF kubernetes conformance verification](https://www.cncf.io/certification/software-conformance/).
 
-> **Important:** If you have existing clusters, please refer to [ks-installer (Install KubeSphere on existing Kubernetes cluster)](https://github.com/kubesphere/ks-installer).
+There are three scenarios to use KubeKey.
 
-## Motivation
+* Install Kubernetes/k3s only
+* Install Kubernetes/k3s and KubeSphere together in one command
+* Install Kubernetes/k3s first, then deploy KubeSphere on it using [ks-installer](https://github.com/kubesphere/ks-installer)
```

Comment:
I think all `k3s` should be updated to `K3s`.

Reply:
Done